source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/root.tsx

```diff
@@ -31,7 +32,8 @@ export default class RootPage extends React.Component {
         <section className='flex flex-grow'>
           <Switch>
             <Route path={loginPath} component={LoginRoute} />
-            <Route path={adminPath} component={AdminRoute} />
+            <Route exact path={adminPath} component={AdminRoute} />
+            <Route path={adminSettingsPath} component={AdminSettingsRoute} />
```

Comment:
nested paths should be within the root admin route

Reply:
changed